packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -91,57 +91,54 @@ export interface WizardProps {
   ariaLabelCloseButton?: string;
 }
 
-export class Wizard extends React.Component<WizardProps> {
-  private static currentId = 0;
-  private static defaultProps = {
+interface WizardState {
```

Comment:
I am new toTypescript. I don't understand why this is needed?

Reply:
We need to define the structure of `state` so TypeScript knows at compile time what the `this.state` object will look like. We pass it like `extends React.Component<WizardProps, WizardState>`.